faqs.html

```diff
@@ -243,7 +235,7 @@ <h1 class="animated fadeInDown">F.A.Q</h1>
                         <p><span>A.</span>- All tickets are transferable.</p>
                         <p class="tg-offset">- Early bird tickets cannot be canceled. They can only be transferred.</p>
                         <p class="tg-offset">- All other tickets can be canceled till 15th Oct 2017.</p>
-                        <p class="tg-offset">- Tickets can be transferred till 31st Oct 2017 using the ticket transfer <a href="https://www.explara.com/manage-tickets">link</a>.</p>
+                        <p class="tg-offset">- Tickets can be transferred till 15th Oct 2017 using the ticket transfer <a href="https://www.explara.com/manage-tickets">link</a>.</p>
```

Comment:
I think this date is no longer valid. We're keeping transfers open till 31st October 11:59 pm. (or correct me if I am wrong, @akshayaurora )

Reply:
Yes, it was discussed and I'm sending a commit to resolve it.